types/passport-42/index.d.ts

```diff
@@ -0,0 +1,94 @@
+// Type definitions for passport-42 1.2
```

Comment:
The format of the repo has changed; you will want to update this package to reflect the new layout. dts-gen 0.7.1 has this fixed already.